### PR TITLE
Feat/veeam

### PR DESCRIPTION
**Pull request by frenautvh**
ref: MANAGER-4017

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report installation#7238.0 ■   
* Stage 1
  * Install ■

File: packages/manager/modules/veeam/src/template.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<div class="cui-page">
+<div class="cui-page veam">
```
Comment (antleblanc) [inline packages/manager/modules/veeam/src/template.html]:
```suggestion
<div class="cui-page veeam">
```

File: packages/manager/modules/veeam/src/storage/add/veeam-storage-add.controller.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+export default class VeeamStorageAddCtrl {
+  constructor($uibModalInstance, serviceName, VeeamService) {
```
Comment (antleblanc) [inline packages/manager/modules/veeam/src/storage/add/veeam-storage-add.controller.js]: Missing ngInject annotation:

```suggestion
  /* @ngInject */
  constructor($uibModalInstance, serviceName, VeeamService) {
```

**Comment by antleblanc** [comment]
> Your README files are empty

I introduced a blank `README.md` file when you want to scaffold a new application/module because we used to copy/paste some other `README.md` but the content wasn't correctly aligned with the application/module itself (e.g. documentation references).

I'm actually working on a template for the `README.md` files for both modules and applications. So in the next weeks or so, theses empty `README.md` files will be updated in a separate Pull Request.

But here we are talking about private packages, so it's not a requirement.

Hope it's fine for you.

**Comment by antleblanc** [comment]
Regarding the API that serve this product, I would prefer name it as `veeam-cloud-connect`.

Is that make sense for you?

**Comment by frenautvh** [comment]
> Regarding the API that serve this product, I would prefer name it as `veeam-cloud-connect`.
> 
> Is that make sense for you?

renamed the module : https://github.com/ovh/manager/pull/1788/commits/abf7bf9f207cc652797c056ef79edf90f6509803

**Pull request merged**